### ISSUE TITLE
Implement react-router support for tab-switcher

### ISSUE THREAD

**Issue by alanmoo** [mozilla/mofo-ui#45]
Is react-router a dependency we should add to Mofo-UI? @gvn?

**Comment by gvn** [comment]
I don't think so, no. TabSwitcher exposes enough of an API that you can integrate with React Router in a project should you choose to. That's what I did on LMO for example.

Not all projects use (or need) RR, and I'd like to keep this library as un-frameworky and lightweight as possible.

**Comment by alanmoo** [comment]
It might be helpful to add some documentation in as to how to do that; I get not wanting to add react-router, but I think using tabs with routes is a common requirement.

**Comment by gvn** [comment]
Agreed. We really need to add documentation to the components in general!

**Comment by gvn** [comment]
Adding to #30
